static/app/components/events/interfaces/breadcrumbs/breadcrumbs.tsx

```diff
@@ -330,7 +330,7 @@ const PanelDragHandle = styled('div')`
     content: '';
     height: 5px;
     width: 100%;
-    border-radius: ${p => p.theme.borderRadiusBottom};
+    border-radius: ${p => p.theme.borderRadius};
```

Comment:
I think the real fix here is to fix the alignment so that it's correctly aligned to the bottom of the panel.

Not sure what we regressed to cause it to be pushed so far down

Reply:
So the bottom corners should be rounded, but remove the gap?

Reply:
yeah, it should be aligned to the bottom of the panel so the bottom edge is aligned with the border